authorization.spec.ts

```diff
@@ -45,39 +51,153 @@ describe("pax2pay.Authorization", () => {
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to succeed",
+	},
+	{
+		amount: ["USD", 801],
+		merchant: {
+			name: "paxair",
+			id: "abcd1234",
+			category: "4511",
+			country: "GB",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to succeed",
+	},
+]
+
+const fails: Omit<pax2pay.Authorization.Creatable, "card" | "reference">[] = [
+	{
+		amount: ["USD", 1],
+		merchant: {
+			name: "paxair",
+			id: "abcd1234",
+			category: "4511",
+			country: "KP",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
+	},
+	{
+		amount: ["USD", 2000],
+		merchant: {
+			name: "paxair",
+			id: "abcd1234",
+			category: "4511",
+			country: "GB",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
 	},
-	acquirer: {
-		id: "2345erty",
-		number: "1351858913568",
-		country: "GB",
+	{
+		amount: ["USD", 2600],
+		merchant: {
+			name: "paxair",
+			id: "abcd1234",
+			category: "4511",
+			country: "GB",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
 	},
-	description: "An upcheck test authorization, to succeed",
-}
-const failedCreatable: Omit<pax2pay.Authorization.Creatable, "card" | "reference"> = {
-	amount: ["USD", 1],
-	merchant: {
-		name: "Merchant",
-		id: "abcd1234",
-		category: "4511",
-		country: "KP",
-		city: "upcheck town",
-		zip: "12345",
-		address: "Streetname 1, 12345 Towncity",
+	{
+		amount: ["USD", 15],
+		merchant: {
+			name: "paxsino",
+			id: "abcd1234",
+			category: "7801",
+			country: "GB",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
 	},
-	acquirer: {
-		id: "2345erty",
-		number: "1351858913568",
-		country: "GB",
+	{
+		amount: ["USD", 15],
+		merchant: {
+			name: "pax dog and horse racing",
+			id: "abcd1234",
+			category: "7802",
+			country: "GB",
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
```

Comment:
Could we write why they will fail in the description? It's hard to know which one should trigger what.

Reply:
true!